packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- Add `submittedTime` to transaction meta ([#1645](https://github.com/MetaMask/core/pull/1645))
+- Add support for actionId ([#1582](https://github.com/MetaMask/core/pull/1582))
```

Comment:
```suggestion
- Add optional `actionId` argument to `addTransaction` and `speedUpTransaction` to prevent duplicate requests. ([#1582](https://github.com/MetaMask/core/pull/1582))
```